src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import { Header, Footer } from './layout';
+import { Repositories } from "./components";
+import './style.css'
+
+export const App = () => {
+    const [formData, setFormData] = useState("");
+    const [username, setUsername] = useState("");
+    const [isSubmitted, setIsSubmitted] = useState(false);
+
+    const handleInput = e => {
+        setFormData(e.target.value)
+    }
+    const handleFormSubmit = e => {
+        e.preventDefault();
+        setUsername(formData);
+        setIsSubmitted(true);
+    }
+
+    return (
+        <>
+            <Header />
```

Comment:
Since the header is used on the results page as well, it might have been alright to keep it in here instead of making a 'Header' component and importing it into this.